style.css

```diff
@@ -656,20 +651,17 @@ footer {
   }
 
   .programs {
-    display: flex;
-    flex-direction: row;
-    margin-top: 3.5rem;
-    flex-wrap: wrap;
-    justify-content: space-around;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
```

Comment:
- [ ] Great job fixing the previous reviewer's request, however, there seems to be a minor problem with this section, for smaller desktop view starting from ~918px this section doesn't wrap with the container instead it gets out.

![image](https://user-images.githubusercontent.com/59416136/157662586-74656bbe-bfc1-42e7-8745-ee8aff3dc591.png)

**Kindly fix this, so that it won't get out of the container.**